roles/keystone-setup/tasks/main.yml

```diff
@@ -42,6 +42,7 @@
                  tenant={{ item.tenant }}
                  token={{ admin_token }}
   with_items: keystone.users
+  when: "{{ item.name }}.enabled is undefined or {{item.name}}.enabled|bool"
```

Comment:
@j2sol "{{item.name}}.enabled|default('true')|bool" 

The syntax above works. However, it needs {{ item }} defined. So since keystonev3 is not in defaults.yml, the syntax will error out for that. So, empty keystonev3 dict might need to be added to defaults.yml

Thoughts on this?